unstructured_inference/inference/layout.py

```diff
@@ -262,15 +265,33 @@ def get_elements_with_detection_model(
             raise ValueError("Invalid OCR mode")
 
         if self.layout is not None:
+            kwargs = {}
```

Comment:
```suggestion
            threshold_kwargs = {}
```
a bit of a nit, but let's make it clear what this kwargs variable is for rather than the kitchen sink